CHANGES.md

```diff
@@ -1,4 +1,11 @@
-# NEXT
+# 2.11.0
+
+## Language changes
+
+* The `newtype` construct, which has existed in the interpreter in an
```

Comment:
Perhaps worth mentioning that the `cryptol-remote-api` doesn't really support newtypes yet.  There's some questions to deal with regarding the generative nature of newtype that haven't fully been worked out yet, unless @pnwamk has done some work in that area I haven't noticed.